src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/RoleAssignment.java

```diff
@@ -22,4 +23,17 @@
     private Instant created;
     private List<String> authorisations;
     private RoleAssignmentAttributes attributes;
+
+    /**
+     * This is how the de-serialization works:
+     * Missing caseId -> caseId == null
+     * caseId: null -> Optional.empty()
+     * caseId: "null" -> Optional.empty()
+     * caseId: "12345" -> Optional.of("12345")
+     */
```

Comment:
we can remove this, since the deserialisation rules are not stated on RoleAssignmentAttributesResource